### PR TITLE
Update Pypi description and README

### PR DESCRIPTION
**Pull request by jitseniesen**
Update long description used on the Pypi website. Update README by adopting a more positive tone in funding announcement, moving it below the screen shot, and removing the headings with the badges (shamelessly lifted from spyder-ide/spyder#6923).

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -46,10 +46,8 @@ def get_package_data(name, extlist):
 This is a plugin for the Spyder IDE that integrates popular unit test
 frameworks. It allows you to run tests and view the results.
 
-**Status:**
-This is a work in progress. It is useable, but only the basic functionality
-is implemented at the moment. The plugin currently supports the py.test and nose
-testing frameworks.
+The plugin supports the `unittest` framework in the Python
+standard library and the `py.test` and `nose` testing frameworks.
```
Comment (CAM-Gerlach) [inline setup.py]: FYI, its officially ``pytest`` now

**Comment by CAM-Gerlach** [comment]
Looking good; some minor comments. Imitation is the sincerest form of flattery...

File: setup.py (hunk 1 above)
Comment (jitseniesen) [reply setup.py]: Right. I'll use another PR to change this en-masse; see #117.

File: AUTHORS [hunk 2]
```diff
@@ -0,0 +1,6 @@
+The Spyder Project Contributors are composed of:
+
+* Pierre Raybaut <pierre.raybaut@gmail.com> (Original Spyder author).
+* Carlos Cordoba <ccordoba12@gmail.com> (Current maintainer).
+* All other developers who have contributed to this repository
+  and/or the precursor in the main spyder-ide/spyder repository.
```
Comment (CAM-Gerlach) [inline AUTHORS]: Missing EoL.

File: setup.py (hunk 1 above)
Comment (CAM-Gerlach) [reply setup.py]: :+1:

**Pull request merged**